include/linker/common-rom.ld

```diff
@@ -35,6 +35,18 @@
 		__object_access_end = .;
 	}
 #endif
+
```

Comment:
can we put this code in some header and just include it here avoiding duplication?

Reply:
Used only once. No need to create a new file just for one instance.